src/persistence.ts

```diff
@@ -124,4 +125,35 @@ export interface Persistence {
      * the appropriate action to take.
      */
     conditionalDeleteResource(request: ConditionalDeleteResourceRequest, queryParams: any): Promise<GenericResponse>;
+
+    /*
+     * Start the job for exporting the database into downloadable files. The file format of the downloadable files
+     * is specified by '_outputFormat'. This interface is to support the bulk export request API
+     * https://hl7.org/Fhir/uv/bulkdata/export/index.html#bulk-data-kick-off-request
+     */
+    initiateExport(
+        requesterUserId: string,
+        requestGranularity: ExportRequestGranularity,
+        requestQueryParams: { _outputFormat?: string; _since?: number; _type?: string },
+        transactionTime: number,

```

Comment:
Calling a persistence microservice should only take hundreds of milliseconds. Is that a concern?

Reply:
We might be calling the persistence layer on a legacy environment that might have a higher latency time than AWS/API Gateway. For example, a user might try to set up `fhir-works` routing and server to be on AWS but then connect it to an on premise database by way of an API. There might be a higher latency time given this scenario.  In essence, I think it would be safer to use the `transactionTime` provided by the router so we don't have to worry about a mismatch between requests reaching the persistence layer and the routing layer.

Reply:
ok. That makes sense